beacon-chain/p2p/message_id.go

```diff
@@ -69,12 +70,18 @@ func MsgID(genesisValidatorsRoot []byte, pmsg *pubsub_pb.Message) string {
 // + message.topic + snappy_decompress(message.data))[:20]. Otherwise, set message-id to the first 20 bytes of the SHA256 hash of the concatenation
 // of the following data: MESSAGE_DOMAIN_INVALID_SNAPPY, the length of the topic byte string (encoded as little-endian uint64),
 // the topic byte string, and the raw message data: i.e. SHA256(MESSAGE_DOMAIN_INVALID_SNAPPY + uint_to_bytes(uint64(len(message.topic))) + message.topic + message.data)[:20].
-func altairMsgID(pmsg *pubsub_pb.Message) string {
+func altairMsgID(pmsg *pubsub_pb.Message, fEpoch types.Epoch) string {
```

Comment:
At the high level, this seems like breaking good function composition especially one as we are setting merge gossip size inside altair msg ID function

Reply:
@terencechain  
I agree. There are two alternatives for this
1) copy the entire function and call it mergeMsgID
2) since MsgID alteration is same for both altair and merge, we should rename the function to something generic like "postAltairMsgID"

I am leaning for the second. what do you think?

Reply:
changed to `postAltairMsgID` per discord discussion with @terencechain